dev/com.ibm.ws.threading/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -21,7 +21,7 @@ name=Name
 name.desc=The name of the Liberty kernel default executor.
 
 core.threads=Core threads
-core.threads.desc=Steady-state or core number of threads to associate with the executor. The number of threads associated with the executor will quickly grow to this number. If this value is less than 0, a default value is used. This default value is calculated based on the number of hardware threads on the system.
+core.threads.desc=Baseline or minimum number of threads to associate with the executor. The number of threads associated with the executor will quickly grow to this number. If this value is less than 0, a default value is used. This default value is calculated based on the number of hardware threads on the system. The minimum number of default executor threads Liberty will use is 4.
```

Comment:
Should you also replace,
"If this value is less than 0, a default value is used."
with
"If this value is less than 4, a default value is used."
given that your changes amount to using a default value (of 4) when the user specifies a value such as 2?

Reply:
Well if the value is less than 0, our default is different ... 

coreThreads config          value Liberty uses
============         ==============
x < 0 (or no config)        Math.max(4, 2 * cpus)
 0 < x < 4                                     4
    x >= 4                                      x

I'm not sure how complicated we want to make the description ...

Reply:
I understand the default is computed differently depending, and that does add complexity, but the statement that "If this value is less than 0, a default value is used." will cause most users to imply that a default value is not used otherwise.  Technically the statement isn't untrue, but it is misleading and I think we can do better.

Here's another way it could be rewritten instead,
"... If this value is less than 0, a default value is calculated based on the number of hardware threads on the system. If the value is a positive number less than or equal to 4, then a default value of 4 is used."

Reply:
Yup, those words make sense, thanks Nate. Maybe I'll shorten the last sentence by dropping 'or equal to', since if the config is 4 then we will use the config.